packages/wonder-blocks-data/util/no-cache.test.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+import NoCache from "./no-cache.js";
+
+import type {IRequestHandler} from "./types.js";
+
+describe("NoCache", () => {
+    describe("#store", () => {
+        it("should not throw", () => {
+            // Arrange
```

Comment:
I wonder if there might be value in calling clone on the data being cached even if we don't cache it so that there cloning errors are consistent between caches.  wdyt?

Reply:
Hmmm, not sure. Seems like it would be allocating memory unnecessarily and adding to the memory burden that has to be garbage collected.